lib/service.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+OOB_URI = 'urn:ietf:wg:oauth:2.0:oob'.freeze
+TOKEN_PATH = 'token.yaml'.freeze
+SCOPE = Google::Apis::CalendarV3::AUTH_CALENDAR_READONLY
+
+def authorize
```

Comment:
I suppose we could make this method private, since its only user is the service in this same class. Do we want to allow external callers to use `authorize`?